### PR TITLE
Update DaysUntilBirthday instructions for macOS sample

### LINKED ISSUE #192: DaysUntilBirthday sample app requires keychain entitlement with Development certificate
Per [this conversation](https://github.com/google/GoogleSignIn-iOS/pull/191#discussion_r939049720) on #191 adding async/await support to the Swift sample app, we discovered that the `keychain-access-groups` entitlement is needed to sign in successfully. We should explore further why this is the case and if we can get around the issue.

### PR DESCRIPTION
**Pull request by mdmathias**
Fixes #192

### CODE REVIEW

File: Samples/Swift/DaysUntilBirthday/README.md [hunk 1]
```diff
@@ -26,6 +26,25 @@ open DaysUntilBirthday.xcodeproj
...
+```
+
+This error is related to the signing of the sample app.
+You have two choices to remedy the problem.
+
+1. We suggest that you manually manage the signing of the macOS sample app so
+that you can provide a provisioning profile. Make sure to select a profile
+that is able to sign macOS apps.
+2. If you do not have the correct provisioning profile, and are unable to
+generate one, then you can add the `keychain-access-groups` entitlement to the
```
Comment (petea) [inline Samples/Swift/DaysUntilBirthday/README.md:45]: I believe this entitlement is presented as the "Keychain Sharing" Capability in Xcode.  That may be a more intuitive way of referring to it.

File: Samples/Swift/DaysUntilBirthday/README.md [hunk 2]
```diff
@@ -26,6 +26,25 @@ open DaysUntilBirthday.xcodeproj
...
+`DaysUntilBirthday (macOS)` target.
```
Comment (petea) [inline Samples/Swift/DaysUntilBirthday/README.md:46]: "... target as a workaround."

File: Samples/Swift/DaysUntilBirthday/README.md [hunk 3]
```diff
@@ -26,6 +26,25 @@ open DaysUntilBirthday.xcodeproj
...
+```
+
+This error is related to the signing of the sample app.
+You have two choices to remedy the problem.
+
+1. We suggest that you manually manage the signing of the macOS sample app so
+that you can provide a provisioning profile. Make sure to select a profile
+that is able to sign macOS apps.
+2. If you do not have the correct provisioning profile, and are unable to
+generate one, then you can add the "Keychain Sharing" entitlement to the
```
Comment (Alex-4-Git) [inline Samples/Swift/DaysUntilBirthday/README.md:45]: Do you want them to follow this one?
https://developer.apple.com/documentation/xcode/configuring-keychain-sharing
If so we can add this link.

Comment (mdmathias) [reply Samples/Swift/DaysUntilBirthday/README.md]: Good suggestion.

**Review by Alex-4-Git — APPROVED** [review]

File: Samples/Swift/DaysUntilBirthday/README.md [hunk 4]
```diff
@@ -26,6 +26,25 @@ open DaysUntilBirthday.xcodeproj
...
+```
+
+This error is related to the signing of the sample app.
+You have two choices to remedy the problem.
+
+1. We suggest that you manually manage the signing of the macOS sample app so
+that you can provide a provisioning profile. Make sure to select a profile
+that is able to sign macOS apps.
+2. If you do not have the correct provisioning profile, and are unable to
+generate one, then you can add the ["Keychain Sharing" entitlement](https://developer.apple.com/documentation/xcode/configuring-keychain-sharing)
```
Comment (petea) [inline Samples/Swift/DaysUntilBirthday/README.md:45]: "entitlement" -> "capability"

**Review by petea — APPROVED** [review]

**Pull request merged**